### PR TITLE
[SITES-445] Correctly determine who sees the review buttons

### PR DESCRIPTION
**Pull request by tessereth**

### CODE REVIEW

File: app/helpers/submissions_helper.rb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 module SubmissionsHelper
   def can_review?
```
Comment (freshtonic) [inline app/helpers/submissions_helper.rb]: Is this helper necessary? I _think_ you could use the callback style of writing the ability and do the `submission.section.present?` logic inside the block.

Comment (tessereth) [inline app/helpers/submissions_helper.rb]: That's a fair point. Will move the logic into `ability.rb`.

Comment (tessereth) [inline app/helpers/submissions_helper.rb]: done

**Comment by maxious** [comment]
LGTM

**Comment by ltankey** [comment]
LGTM

**Pull request merged**